python/ray/data/aggregate/_aggregate.py

```diff
@@ -27,32 +54,6 @@ def __init__(
         finalize: Callable[[AggType], U] = lambda a: a,
         name: Optional[str] = None,
     ):
-        """Defines an aggregate function in the accumulator style.
```

Comment:
should we still keep the args here, so that it shows on the `__init__` page?

Reply:
There won't be an `__init__` page after this (it'd be redundant)
<img width="1204" alt="image" src="https://github.com/ray-project/ray/assets/26107013/4438d314-11d5-4d77-8548-56d4d8e5ea43">